bot.js

```diff
@@ -1,14 +1,12 @@
 const Discord = require('discord.js');
 const client = new Discord.Client();
-const dotenv = require('dotenv');
 const movie = require('./modules/movie-fetch.js');
 
-dotenv.config();
```

Comment:
Do not we need it?

Reply:
I import the config via command line so that we don't have to clean up the code when we deploy on to Heroku.